juniper/src/executor_tests/variables.rs

```diff
@@ -5,30 +5,38 @@ use crate::{
     schema::model::RootNode,
     types::scalars::{EmptyMutation, EmptySubscription},
     validation::RuleError,
-    value::{DefaultScalarValue, Object, ParseScalarResult, ParseScalarValue},
+    value::{DefaultScalarValue, Object},
     GraphQLError::ValidationError,
-    GraphQLInputObject,
+    GraphQLInputObject, InputValue, ScalarValue, Value,
 };
 
 #[derive(Debug)]
 struct TestComplexScalar;
 
-#[graphql_scalar]
-impl<S: ScalarValue> GraphQLScalar for TestComplexScalar {
-    fn resolve(&self) -> Value {
+#[graphql_scalar(
+    name = "TestComplexScalar",
```

Comment:
Let's makr such places with `// TODO` comment to not forget restore it to `derive` variant in the next PR.